.github/workflows/uploadDockerImage.yml

```
@@ -1,30 +1,19 @@
-name: Upload tag to GitHub
-
+name: Publish Docker image
 on:
-  push:
-    tags:
-      - '*'
-
+  release:
+    types: [published]
 jobs:
-  build:
-    runs-on: ubuntu-latest
-
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v2
-    
-    - name: Export tag version
-      run: echo ::set-env name=VERSION::${GITHUB_REF/refs\/tags\//}
-    
-    - name: Docker Login
-      run: echo $PACKAGES_WRITE_TOKEN | docker login docker.pkg.github.com -u $USERNAME --password-stdin
-      env:
-        PACKAGES_WRITE_TOKEN: ${{ secrets.DOCKER_GITHUB_PASSWORD }}
-        USERNAME: ${{ secrets.DOCKER_GITHUB_USERNAME }}
-    - name: Docker Build
-      run: docker build -t docker.pkg.github.com/danger/danger-js/danger-danger-js:$VERSION .
-      env:
-        USERNAME: ${{ secrets.DOCKER_GITHUB_USERNAME }}
-    - name: Deploy
-      run: docker push docker.pkg.github.com/danger/danger-js/danger-danger-js:$VERSION
-      env:
-        USERNAME: ${{ secrets.DOCKER_GITHUB_USERNAME }}
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to GitHub Packages
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          registry: docker.pkg.github.com
+          repository: danger/danger-js/danger-danger-js
```

I think this should be 

```suggestion
          registry: ghcr.io/danger
          repository: danger-js
```

But I'm not familiar with this action